src/Component/Symbolizer/Editor/Editor.tsx

```diff
@@ -95,7 +97,7 @@ export class Editor extends React.Component<EditorProps, EditorState> {
     }
   }
 
-  getUiFromSymbolizer = (symbolizer: Symbolizer): React.ReactNode => {
+  getUiFromSymbolizer = (symbolizer: Symbolizer, composition: Compositions): React.ReactNode => {
```

Comment:
Please add some Code docs to this and the other newly introduced functions.

Reply:
solved in https://github.com/geostyler/geostyler/pull/1186/commits/587b9b93e34174a44cce9a8464bcb50512687670